src/features/posts/components/add-post-form.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { PostContent } from '../types/post-content';
+import { postContentSchema } from '../utils/validate';
+import { fromZodError } from 'zod-validation-error';
+
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import { Button } from '@mui/material';
+
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+
+
+let DEFAULT_POST: PostContent = {
```

Comment:
czemu let?

Reply:
To tak samo jak wcześniej, dlatego ze mi się wydawało ze się zmieniała, ale jednak nie